packages/features/bookings/components/event-meta/AvailableEventLocations.tsx

```diff
@@ -22,7 +22,8 @@ function RenderIcon({
     <img
       src={eventLocationType.iconUrl}
       className={classNames(
-        eventLocationType?.iconUrl?.includes("-dark") && "dark:invert",
+        (eventLocationType?.iconUrl?.includes("-dark") || eventLocationType?.iconUrl === "/map-pin.svg") &&
+          "dark:invert",
```

Comment:
@Ryukemeister pushed this change as there can be many icons for location depending on the location

Reply:
@Udit-takkar thanks, might've missed that!

Reply:
How about renaming the file to `map-pin-dark.svg`? That way there is no specific file name check here.